packages/react-percy/src/runPercy.js

```diff
@@ -1,6 +0,0 @@
-import run from '@percy-io/react-percy-ci';
-
-export default function runPercy(percyConfig, webpackConfig, percyToken) {
-  // Eventually this will switch to running the local dev-server when not in CI
-  return run(percyConfig, webpackConfig, percyToken);
```

Comment:
Just killed this wrapper function. If we have a dev-viewer in the future, it probably ought to be via a completely separate CLI interface like `percy-snapshot-dev-server`.